docs/serving/configuration/config-defaults.md

```diff
@@ -30,319 +32,497 @@ data:
-By default, it is not set by Knative.
+The `revision-cpu-request` value determines the CPU allocation assigned to revisions by default. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-cpu-request`
-
-**Default**: `"400m"` (0.4 of a CPU (aka 400 milli-CPU))
+* **Global key:** `revision-cpu-request`
+* **Per-revision annotation key:** `serving.knative.dev/revision-cpu-request`
+* **Possible values:** integer
+* **Default:** `"400m"` (0.4 of a CPU, or 400 milli-CPU)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-cpu-request: "400m" # 0.4 of a CPU (aka 400 milli-CPU)
-```
 
-### Revision Memory Request
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-cpu-request: "400m"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-cpu-request: "400m"
+        spec:
+          containers:
+            - image: gcr.io/knative-samples/helloworld-go
+    ```
+
+### Revision memory request
 {% raw %}
-revision-memory-request contains the memory allocation to assign
-to revisions by default.  If omitted, no value is specified
-and the system default is used.
-Below is an example of setting revision-memory-request.
-By default, it is not set by Knative.
+The `revision-memory-request` value determines the memory allocation assigned
+to revisions by default. If this value is omitted, the system default is used. This key is not enabled by default for Knative.
 {% endraw %}
 
-**Key**: `revision-memory-request`
-
-**Default**: `"100M"` (100 megabytes of memory)
+* **Global key:** `revision-memory-request`
+* **Per-revision annotation key:** `serving.knative.dev/revision-memory-request`
+* **Possible values:** integer
+* **Default:** `"100M"` (100 megabytes of memory)
 
 **Example:**
-```yaml
-apiVersion:  v1
-kind:  ConfigMap
-metadata:
-  name:  config-defaults
-  namespace:  knative-serving
-data:
-  revision-memory-request: "100M" # 100 megabytes of memory
-```
+
+=== "Global (ConfigMap)"
+    ```yaml
+    apiVersion:  v1
+    kind:  ConfigMap
+    metadata:
+      name:  config-defaults
+      namespace:  knative-serving
+    data:
+      revision-memory-request: "100M"
+    ```
+
+=== "Per Revision"
+    ```yaml
+    apiVersion: serving.knative.dev/v1
+    kind: Service
+    metadata:
+      name: helloworld-go
+      namespace: default
+    spec:
+      template:
+        metadata:
+          annotations:
+            serving.knative.dev/revision-memory-request: "400m"
```

Comment:
Likewise this annotation doesn't exist - the config map property configures the  default value for `Spec.Containers[*].Requests.Memory` of a `Revision`

Reply:
similar to above but "memory" instead of "cpu" ie.  https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#example-1